src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -52,6 +52,7 @@ export class ResultComponent implements OnInit, OnChanges {
     @Input() showUngradedResults: boolean;
     @Input() showGradedBadge = false;
     @Input() showTestDetails = false;
+    @Input() correctionRound = 0;
```

Comment:
I don't like this dependency here, please remove it and rather inject a participation with only one result (either first or second correction)